### PR TITLE
RedisCacher pingInterval Option

### PR DESCRIPTION
**Pull request by shawnmcknight**
## :memo: Description

This PR adds a new option `pingInterval` to the `RedisCacher`.  If this option is set, it will send a PING command to the Redis server every `pingInterval` milliseconds.  The intention is for this to keep traffic active on the connection to avoid idle timeouts introduced by services like Azure Cache for Redis.  

### :dart: Relevant issues

N/A

### :gem: Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## :vertical_traffic_light: How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Tested in production environment as well as new unit tests.

## :checkered_flag: Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] **I have added tests that prove my fix is effective or that my feature works**
- [X] **New and existing unit tests pass locally with my changes**
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/cachers/redis.js [hunk 1]
```diff
@@ -114,6 +117,19 @@ class RedisCacher extends BaseCacher {
 		this.serializer = Serializers.resolve(this.opts.serializer);
 		this.serializer.init(this.broker);
 
+		// add interval for ping if set
+		if (typeof this.opts.pingInterval === "number") {
```
Comment (icebob) [inline src/cachers/redis.js:121]: Would be to skip the type checking. Just check `this.opts.pingInterval > 0` and in `setInterval` use `Number(this.opts.pingInterval). The reason is that if you try to configure it via environment variables (MOL_CACHER__OPTIONS__PING_INTERVAL=5000), the value will be `String`

**Review by icebob — CHANGES_REQUESTED** [review]

File: src/cachers/redis.js (hunk 1 above)
Comment (shawnmcknight) [reply src/cachers/redis.js]: Good call.  I've made the proposed change and added a few more tests for it.

**Review by icebob — APPROVED** [review]
Good job, thanks!

**Comment by icebob** [comment]
It has been released. @shawnmcknight could you help me with the docs updating?
https://github.com/moleculerjs/site/blob/master/source/docs/0.14/caching.md#redis-cacher

**Comment by shawnmcknight** [comment]
> It has been released. @shawnmcknight could you help me with the docs updating?
> https://github.com/moleculerjs/site/blob/master/source/docs/0.14/caching.md#redis-cacher

Created https://github.com/moleculerjs/site/pull/154

**Pull request merged**